build.sbt

```diff
@@ -5,7 +5,7 @@ name := "sigma-state"
 lazy val commonSettings = Seq(
   organization := "org.scorexfoundation",
   version := "0.9.4-SNAPSHOT",
-  scalaVersion := "2.12.6",
+  scalaVersion := "2.12.4",
   licenses := Seq("CC0" -> url("https://creativecommons.org/publicdomain/zero/1.0/legalcode")),
```

Comment:
@kushti why ?

Reply:
With latest versions of Scala & SBT, the latter hangs sometimes after "Done compiling" message (and not on my machine only, reported also by Leo). I dunno what is causing this though, Scala or SBT.